src/server/typingsInstaller/typingsInstaller.ts

```diff
@@ -41,13 +43,74 @@ namespace ts.server.typingsInstaller {
         onRequestCompleted: RequestCompletedAction;
     }
 
+    interface TypeDeclarationTimestampFile {
+        entries: MapLike<number>;

```

Comment:
Always worth having a comment explicitly describing the keys and values of a map (e.g. `This is a map from the package name, e.g. "react", to the timestamp value (as produced by Date#getTime)`)

Reply:
Added.